apps/registry/src/index.ts

```diff
@@ -1,10 +1,7 @@
-import {
-  DurableObjectNamespace,
-  Request,
-  Response,
-} from '@cloudflare/workers-types';
+import { DurableObjectNamespace } from '@cloudflare/workers-types';
```

Comment:
You might need to still import Request and Response here because they differ slightly from the default types inside the worker environment.